examples/Thermostat/slice/ThermoControl.slice

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) ZeroC, Inc.
+
+module Igloo
+
+/// Controls the actual thermostat.
```

Comment:
This comment seems to vaguely reference other interfaces that maybe control a not actual thermostat. 
```suggestion
/// Controls a physical thermostat device.
```